examples/typescript-plugin/src/strapi-server.ts

```diff
@@ -0,0 +1,3 @@
+import services from './services';
+
+export = () => ({ services });
```

Comment:
export default

Reply:
It won't work since it'll transpile to `exports.default = () => ({ ... });` instead of `module.exports = () => ({ ... })`

Reply:
What you usually do is a `require('./server').default;` in the entrypoint and an export default in the server/index.ts